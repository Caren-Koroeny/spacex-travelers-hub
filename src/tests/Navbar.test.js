import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import { render } from '@testing-library/react';
import Navigation from '../components/Navbar';

describe('Navigation component', () => {
  it('should render Header component', () => {
    const { container } = render(
      <BrowserRouter>
        <Navigation />
      </BrowserRouter>,

    );

    expect(container).toMatchSnapshot();
  });
});
