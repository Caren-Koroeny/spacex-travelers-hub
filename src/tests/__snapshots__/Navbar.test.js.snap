// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation component should render Header component 1`] = `
<div>
  <header
    class="headerSection"
  >
    <section
      class="logoSection"
    >
      <img
        alt="logo"
        class="logo"
        src="planet1.png"
      />
      <h1>
        Space Travelers Hub
      </h1>
    </section>
    <nav
      class="navLinks"
    >
      <a
        aria-current="page"
        class="active"
        href="/"
        style="color: rgb(100, 149, 237); font-weight: 700; text-decoration: underline;"
      >
        Rockets
      </a>
      <a
        class=""
        href="/missions"
        style="font-weight: regular;"
      >
        Missions
      </a>
      <div>
        |
      </div>
      <a
        class=""
        href="/myprofile"
        style="font-weight: regular;"
      >
        My Profile
      </a>
    </nav>
  </header>
  <hr />
</div>
`;
